test/integration/prometheus_endpoint.exs

```diff
@@ -0,0 +1,42 @@
+defmodule MongoosePush.API.ConnectionTest do
+  use ExUnit.Case, async: false
+  use AssertEventually, timeout: 5000, interval: 10
+
+  alias MongoosePush.Support.API, as: Tools
+
+  @url "/v1/notification/f534534543"
+
+  setup do
+    Tools.reset(:fcm)
+    :ok
+  end
+
+  # @tag :skip
```

Comment:
This test is not being picked up by ExUnit because it doesn't have the `_test.exs` suffix. We should rename that file and enable the skip that should be required after the rename.